ovs/extensions/healthcheck/generic_hc.py

```diff
@@ -86,6 +86,27 @@ def get_log_files_by_path(start_path, recursive=True):
             result_handler.success('All log files are ok!')
 
     @staticmethod
+    @expose_to_cli('ovs','port-ranges-test',HealthCheckCLIRunner.ADDON_TYPE)
+    def check_port_ranges(result_handler, requested_number_of_ports=2, minimal_number_of_ports=2):
+        """
+        Checks the port range that is available
```

Comment:
Docstring says nothing
`Checks whether the expected amount of ports is available for the requested amount of ports`